src/ResponseFactory.php

```diff
@@ -9,7 +9,6 @@ class ResponseFactory implements ResponseFactoryInterface
 {
     public function createResponse($code = 200)
     {
-        return (new Response())
-            ->withStatus($code);
+        return new Response('php://memory', $code);
```

Comment:
Just a super minor. If we use the default value for this arguments then there is one less object that will be instantiated.